### PR TITLE
dev/core#708, Fix Qill for Added by and Modified By

### PR DESCRIPTION
**Pull request by pradpnayak**
Overview
----------------------------------------
https://lab.civicrm.org/dev/core/issues/708

Before
----------------------------------------
before

After
----------------------------------------
after

### CODE REVIEW

File: CRM/Contact/BAO/Query.php [hunk 1]
```diff
@@ -3899,7 +3899,21 @@ public function changeLog(&$values) {
...
+    foreach ($this->_params as $params) {
+      if ($params[0] == 'log_date') {
+        if ($params[2] == 2) {
+          $fieldTitle = ts('Modified By');
+        }
+        break;
+      }
+    }
+    list($qillop, $qillVal) = self::buildQillForFieldValue(NULL, 'changed_by', $name, 'LIKE');
+    $this->_qill[$grouping][] = ts("%1 %2 %3", [
```
Comment (monishdeb) [inline CRM/Contact/BAO/Query.php]: @pradpnayak can you  add single quotes for value:
```php
$this->_qill[$grouping][] = ts("%1 %2 '%3'", [
...
```

Comment (pradpnayak) [reply CRM/Contact/BAO/Query.php]: Thanks @monishdeb 

Pushed the change.

Comment (eileenmcnaughton) [reply CRM/Contact/BAO/Query.php]: @monishdeb looks like your review has been responded to - good to merge now?

**Comment by monishdeb** [comment]
Tested, working fine.

**Pull request merged**